nexus/examples/qmcpack/rsqmc_misc/excited/vmc_excitation_alternatives.py

```diff
@@ -0,0 +1,355 @@
+    conv_thr     = 1e-8,
+    nosym        = True,
+    wf_collect   = True,
+    system       = dia,
+    nbnd         = 8,      #a sensible nbnd value can be given
+    verbosity    = 'high', #verbosity must be set to high
+    pseudos      = ['C.BFD.upf'],
+    dependencies = (scf,'charge_density'),
+    )
+
+conv = generate_pw2qmcpack(
+    identifier   = 'conv',
+    path         = 'diamond/nscf',
+    job          = job(cores=1,app='pw2qmcpack.x', hours = 1),
+    write_psir   = False,
+    dependencies = (nscf,'orbitals'),
+    )
+
+opt = generate_qmcpack(
+    identifier      = 'opt',
+    path            = 'diamond/opt',
+    job             = job(cores=16,threads=16,app='qmcpack', hours = 1),
+    input_type      = 'basic',
+    system          = dia,
+    pseudos         = ['C.BFD.xml'],
+    twistnum        = 0,
+    J2              = True,         # Add a 2-body B-spline Jastrow
+    spin_polarized  = True,
+    qmc             = 'opt',        # Do a wavefunction optimization
+    minmethod       = 'oneshift',   # Optimization algorithm (assumes energy minimization)
+    init_cycles     = 4,            # First 4 iterations allow large parameter changes
+    cycles          = 10,           # 8 subsequent iterations with smaller parameter changes
+    warmupsteps     = 8,            # First 8 steps are not recorded
+    blocks          = 100,          # Number of blocks to write in the .scalar.dat file
+    timestep        = 0.4,          # MC step size (nothing to do with time for VMC)
+    init_minwalkers = 0.01,         # Smaller values -> bigger parameter change
+    minwalkers      = 0.5,          #
+    samples         = 5000,         # VMC samples per iteration
+    use_nonlocalpp_deriv = False,
+    dependencies    = (conv,'orbitals'),
+    )
+
+################################################################################
+############ Ground State at Gamma #############################################
+################################################################################
+qmc_ground = generate_qmcpack(
+    det_format     = 'old',
+    identifier     = 'vmc',
+    path           = 'diamond/vmc_ground',
+    job            = job(cores=16,threads=16,app='qmcpack', hours = 1),
+    input_type     = 'basic',
+    spin_polarized = True,
+    twistnum        = 0,
+    system         = dia,
+    pseudos        = ['C.BFD.xml'],
+    jastrows       = [],
+    calculations   = [
+        vmc(
+            warmupsteps =  20,
+            blocks      = 2400,
+            steps       =  25,
+            substeps    =   2,
+            timestep    =  .4,
+            )
+        ],
+    dependencies = [(conv,'orbitals'),
+                    (opt,'jastrow')],
+    )
+
+################################################################################
+############ Single Determinant Excitations ####################################
+################################################################################
+
```

Comment:
Make a note that all the below do the exact same thing and just use alternative syntax.